site/learn/Learn-Execution.md

```diff
@@ -56,7 +58,9 @@ If a field produces a scalar value like a string or number, then the execution c
 
 At the top level of every GraphQL server is a type that represents all of the possible entry points into the GraphQL API, it's often called the *Root* type or the *Query* type.
 
-In this example, our Query type provides a field called `human` which accepts the argument `id`. The resolver function for this field likely accesses a database and then constructs and returns a `Human` object.
+In this example, our Query type provides a field called `human` which accepts the argument `id`. The resolver function for this field likely accesses a database and then constructs and returns a `Human` object. 
+
+Notice that you can also return extra properties that are not part of the instance `Human`, so deeper resolvers in your query resolution tree could take these extra data and use it to help resolve nested information (see **Providing extra data in a resolver** section).
```

Comment:
This is incorrect - a resolver cannot return extra information other than the resolved value